docs/source/loading.rst

```diff
@@ -0,0 +1,304 @@
+    >>> trains_ds = datasets.load_dataset('bookcorpus', [(datasets.ReadInstruction('train', to=k, unit='%') + datasets.ReadInstruction('train', from_=k+10, unit='%')) for k in range(0, 100, 10)])
+
+
+Percent slicing and rounding
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For Datasets where the requested slice boundaries do not divide evenly by 100, the default behavior is to round the boundaries to the nearest integer. As a result, some slices may contain more examples than others as shown in the following example:
+
+.. code-block::
+
+    # Assuming `train` split contains 999 records.
+    # 19 records, from 500 (included) to 519 (excluded).
+    >>> train_50_52_ds = datasets.load_dataset('bookcorpus', split='train[50%:52%]')
+    # 20 records, from 519 (included) to 539 (excluded).
+    >>> train_52_54_ds = datasets.load_dataset('bookcorpus', split='train[52%:54%]')
+
+If you want equal sized splits, use ``pct1_dropremainder`` rounding instead. This will treat the specified percentage boundaries as multiples of 1%. 
+
+.. code-block::
+
+    # 18 records, from 450 (included) to 468 (excluded).
+    >>> train_50_52pct1_ds = datasets.load_dataset('bookcorpus', split=datasets.ReadInstruction( 'train', from_=50, to=52, unit='%', rounding='pct1_dropremainder'))
+    # 18 records, from 468 (included) to 486 (excluded).
+    >>> train_52_54pct1_ds = datasets.load_dataset('bookcorpus', split=datasets.ReadInstruction('train',from_=52, to=54, unit='%', rounding='pct1_dropremainder'))
+    # Or equivalently:
+    >>> train_50_52pct1_ds = datasets.load_dataset('bookcorpus', split='train[50%:52%](pct1_dropremainder)')
+    >>> train_52_54pct1_ds = datasets.load_dataset('bookcorpus', split='train[52%:54%](pct1_dropremainder)')
+
+.. important::
+
+    Using ``pct1_dropremainder`` rounding may truncate the last examples in a dataset if the number of examples in your dataset don't divide evenly by 100.
+
+Troubleshooting
+---------------
+
+Sometimes, loading a dataset is not as simple as calling :func:`dataset.load_dataset`. In this section, you will learn how to solve two of the most common issues you may encounter when you load a dataset: manually download a dataset, and specify features of a dataset.
+
+Manual download
+^^^^^^^^^^^^^^^
+
+Certain datasets will require you to manually download the dataset files due to licensing incompatibility or if the files are hidden behind a login page. This will cause :func:`dataset.load_dataset` to throw an ``AssertionError``. But Datasets provides detailed instructions for downloading the missing files. After you have downloaded the files, use the ``data_dir`` argument to specify the path to the files you just downloaded. For example, if you try to download the PAN-X.fr configuration from the `xtreme dataset <https://sites.research.google/xtreme>`_:
+
+    >>> dataset = load_dataset("xtreme", "PAN-X.fr")
```

Comment:
xtreme no longer requires a manual download, we'll have to update this part

Reply:
do you know of another dataset that requires manual download? we can switch it with xtreme here

Reply:
oh nevermind, found one :)